osfoffline/settings/__init__.py

```diff
@@ -16,4 +19,44 @@
     logger.info('Ensuring {} exists'.format(path))
     os.makedirs(path, exist_ok=True)
 
+
+# Define logging configuration to use individual override params from settings files
+LOGGING_CONFIG = {
+    'version': 1,
+    'disable_existing_loggers': False,
+    'formatters': {
+        'console': {'format': FILE_FORMATTER},
+        'file_log': {'format': FILE_FORMATTER}
+    },
+    'handlers': {
+        'console': {
+            'class': 'logging.StreamHandler',
+            'level': LOG_LEVEL,
+            'formatter': 'console'
+        },
+        'syslog': {
+            'class': 'logging.handlers.SysLogHandler',
+            'level': LOG_LEVEL
+        },
+        'logfile': {
+            'class': 'logging.FileHandler',
+            'level': LOG_LEVEL,
+            'filename': PROJECT_LOG_FILE,
+            'formatter': 'file_log'
+        },
+    },
+    'root': {
+        'level': LOG_LEVEL,
+        'handlers': ['console', 'logfile']
+    }
+}
+
+# Set up basic (console and file) logging
 logging.config.dictConfig(LOGGING_CONFIG)
+
+# Add Sentry logging separately, so that we can access the client and modify context variables later
+# This allows us to send additional data to Sentry (like username, when the user is logged in)
+# TODO: We should allow the user to choose whether they log to sentry
```

Comment:
> Agreed, logged as v1 blocker [#OFF-74].

Would you like me to create a JIRA ticket separate from [#OFF-74] to track followon improvement work?